website/i18n/zh/code.json

```diff
@@ -35,6 +27,86 @@
     "message": "让产品与技术背后的细节更加可视化",
     "description": "We love open source."
   },
+  "download.website.title": {
+    "message": "开启智能网关之旅",
+    "description": "Downloads"
+  },
+  "download.website.subtitle": {
+    "message": "We love open source.",
+    "description": "We love open source."
+  },
+  "download.website.history.title": {
+    "message": "历史版本",
+    "description": "History Versions"
+  },
+  "download.website.history.all": {
+    "message": "如果你想要预览 Apache APISIX 已发布的所有版本，请查看 ",
+    "description": "Find all APISIX releases in the&nbsp;"
+  },
+  "download.website.link.archive": {
+    "message": "Archive repository",
```

Comment:
```suggestion
    "message": "历史版本归档",
```

Reply:
> 

I think the link of the **file repository** should not be translated directly. 
The file repository **path** or **name** corresponds to English, which may cause trouble for developers after translation.

Reply:
OK